doc/ext/sphinxarg/ext.py

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
Is the sphinxarg ext vendored? (https://sphinx-argparse.readthedocs.io/en/stable/install.html)

Suggest to exclude it from reformatting.